src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -92,7 +92,8 @@ public function block($messages, $type = null, $style = null, $prefix = ' ', $pa
             }
         }
 
-        $this->writeln(implode("\n", $lines)."\n");
+        $lines[] = '';
+        $this->writeln($lines);
```

Comment:
@aitboudad : array_push returns the new number of elements in the array. So we will not be able to inline those statements using `array_push`.